skbio/alignment/_tabular_msa_repr.py

```diff
@@ -0,0 +1,52 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+
+from skbio.util._metadata_repr import _MetadataReprBuilder
+
+
+class _TabularMSAReprBuilder(_MetadataReprBuilder):
+
+    def _process_header(self):
+        cls_name = self._obj.__class__.__name__
+        if self._obj.dtype is not None:
+            dtype_class_name = self._obj.dtype.__name__
+        else:
+            dtype_class_name = 'No dtype'
+        self._lines.add_line(cls_name + ' <' + dtype_class_name + '>')
```

Comment:
We were thinking of using parameterized type notation, e.g. similar to C++ and Java:

```java
ArrayList<Integer> numbers = new ArrayList<Integer>();
```

I compared both styles and like it without the space because it matches the style above and seems to make a stronger statement about it being a `TabularMSA` *of* `DNA` sequences.

@ebolyen what's your preference?